CHANGES_UNRELEASED.md

```diff
@@ -2,15 +2,10 @@
 
 # Unreleased Changes
 
-[Full Changelog](https://github.com/mozilla/application-services/compare/v93.0.3...main)
+[Full Changelog](https://github.com/mozilla/application-services/compare/v93.0.4...main)
 
 <!-- WARNING: New entries should be added below this comment to ensure the `./automation/prepare-release.py` script works as expected.
 
-## Places
```

Comment:
uh looks like this didn't get copied properly by the script, I'll do it manually